build/transpile.js

```diff
@@ -924,25 +924,26 @@ class Transpiler {
         try {
 
             const { python2Folder, python3Folder, phpFolder } = options
-
-            const jsmtime = fs.statSync (jsFolder + filename).mtime
-            const python2mtime = fs.statSync (python2Folder + filename.replace ('.js', '.py')).mtime
-            const python3mtime = fs.statSync (python3Folder + filename.replace ('.js', '.py')).mtime
-            const phpmtime = fs.statSync (phpFolder + filename.replace ('.js', '.php')).mtime
+            const pythonFilename = filename.replace ('.js', '.py')
+            const phpFilename = filename.replace ('.js', '.php')
+            const jsmtime = fs.statSync (jsFolder + filename).mtime.getTime ()
+            const python2mtime = fs.statSync (python2Folder + pythonFilename).mtime.getTime ()
+            const python3mtime = fs.statSync (python3Folder + pythonFilename).mtime.getTime ()
+            const phpmtime = fs.statSync (phpFolder + phpFilename).mtime.getTime ()
             const contents = fs.readFileSync (jsFolder + filename, 'utf8')
 
-            if (jsmtime.getTime() != python2mtime.getTime() || jsmtime.getTime() != python3mtime.getTime() || jsmtime.getTime() != phpmtime.getTime()) {
```

Comment:
did you test this ? because from my testing it always thinks the file has changed if you do this